admin/tools/docker/3_build-tagged-image.sh

```diff
@@ -57,31 +56,26 @@ if [ $# -ne 1 ] ; then
     exit 2
 fi
 
-MASTER=$1
+GIT_TAG=$1
 
 DIR=$(cd "$(dirname "$0")"; pwd -P)
-DOCKERDIR="$DIR/cfg"
-
-# Build the master image
+DOCKERDIR="$DIR/tagged"
 
-sed 's/{{NODE_TYPE_OPT}}/-m/g' cfg/Dockerfile.tpl | \
-    sed "s/{{MASTER_FQDN_OPT}}/${MASTER}/g" > $DOCKERDIR/Dockerfile
-
-VERSION=master-${MASTER}
-TAG="fjammes/qserv:$VERSION"
-printf "Building development image %s from %s\n" "$TAG" "$DOCKERDIR"
-docker build --tag="$TAG" "$DOCKERDIR"
-
-printf "Image %s built successfully\n" "$TAG"
 
-# Build the worker image
+# Build the image
 
-sed 's/{{NODE_TYPE_OPT}}//g' cfg/Dockerfile.tpl | \
-    sed "s/{{MASTER_FQDN_OPT}}/${MASTER}/g" > $DOCKERDIR/Dockerfile
+TIMESTAMP=$(date --utc)
+DOCKERFILE="$DOCKERDIR/Dockerfile"
+cp "$DOCKERDIR/Dockerfile.tpl" "$DOCKERFILE"
+sed -i "s%{{GIT_TAG_OPT}}%${GIT_TAG}%g" "$DOCKERFILE"
+# Force the build of the branch by changing timestamp
+sed -i "s%{{TIMESTAMP}}%${TIMESTAMP}%g" "$DOCKERFILE"
```

Comment:
Would invoking `docker build` with `--no-cache=true` be a viable alternative to this?